test/e2e_kubeadm/bootstrap_token_test.go

```diff
@@ -34,6 +35,21 @@ const (
 	bootstrapTokensCSRAutoApprovalClusterRoleName    = "system:certificates.k8s.io:certificatesigningrequests:nodeclient"
 )
 
+var (
+	bootstrapTokensAllowPostCSR = &authv1.ResourceAttributes{
+		Resource: "certificatesigningrequests",
+		Group:    "certificates.k8s.io",
+		Verb:     "create",
+	}
+
+	bootstrapTokensCSRAutoApproval = &authv1.ResourceAttributes{
+		Resource:    "certificatesigningrequests",
+		Subresource: "nodeclient",
+		Group:       "certificates.k8s.io",
+		Verb:        "create",
```

Comment:
the `system:certificates.k8s.io:certificatesigningrequests:nodeclient` cluster role does only have the `create` verb.
we already check for the related cluster role and binding.
deferring to @fabriziopandini again.